.travis.yml

```diff
@@ -39,4 +39,8 @@ jobs:
       after_success: skip
 
 after_success:
-  - julia -e 'using Pkg; cd(Pkg.dir("LinearOperators")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(process_folder())'
+  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.2"
+      using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(process_folder())
+    else
+      println("Coverage skipped")
```

Comment:
Is the `else` necessary?

Reply:
No, I can just not print anything if the coverage is skipped, should I change it?

Reply:
If you don't mind, yes. It seems simpler. That's what we do for the docs.

Reply:
Actually, is it possible to request coverage as a `job` in the `travis.yml`?

Reply:
If I request as a job it will have to redo the tests it just did, so I chose not to.

Reply:
Done.

Reply:
I don't understand the problem with a job. Doesn't it mean that only one VM will run the coverage test instead of all of them launching Julia to check if it's the right VM?

Reply:
The `after_success` is run after the tests, inside the environment that was used before. The `job` will be run as a separate instance, after all tests have been run, and it will have to run `Pkg.test()` again, because it needs the coverage information. It's completely doable, the only problem is that it repeats the work of an instance.

Reply:
Got it. Thanks for explaining.